.github/workflows/ci.yml

```diff
@@ -389,3 +389,4 @@ jobs:
           project_name: relay
           local_image: ${{ steps.image_pull.outputs.relay-test-image-name }}
           docker_repo: docker.io/getsentry/relay
+          docker_password: ${{ secrets.DOCKERHUB_PASSWORD }}
```

Comment:
should the action-self-hosted-e2e-tests@5bbd3d6725bbe4bf5cc12f8a3291ce87df6a891f be the newest commit?

Reply:
Yes, I should update it to 05814e09c6cc1d9d1ac1c03a22216b7fbed1756a actually.